tests/main/interfaces-kernel-module-load/test-snapd-kernel-module-load/meta/snap.yaml

```diff
@@ -8,10 +8,10 @@ plugs:
         modules:
             - name: mymodule
               load: denied
-            - name: parport_pc
+            - name: bfq
               load: on-boot
-              options: io=0x3bc,0x278 irq=none
+              options: slice_idle_us=20 strict_guarantees=1
             - name: other_module
               load: denied
-            - name: pcspkr
+            - name: arc4
```

Comment:
Why these changes, is this because these modules are common for all platforms?

Reply:
Yeah, maybe worth to extend the commit message here a bit so that it includes information that one of the current modules crashes the kernel and the other is not available?

Reply:
I hope they are :-) The problem was that on the PI the `pcspkr` module was missing, and the `parport_pc` was causing a (recoverable) kernel crash.